SonarQube.Bootstrapper/SonarQube.Analysis.xml

```diff
@@ -5,16 +5,16 @@
   is located (if it exists). It is possible to use another properties file by using the /s:filePath.xml flag
   
   The overriding strategy of property values is the following:
-  - A property defined in the command line (/d:propertyName=value) has the top priority and so can override :
-  - A property defined in an MSBuild project file (corresponding to a SonarQube module) which can override :
-  - A property defined in the MSBuild.Analysis.xml configuration file [this file] which can override :
+  - A project-specific property defined in the MSBuild *.*proj file (corresponding to a SonarQube module) can override:
```

Comment:
fixed @bogdang83 

This has always been the de facto ordering: The SonarQube runner gives a higher precedence to module specific properties, so `{ProjectGuid}.sonar.foo=...` will take precedence over the global `sonar.foo=...` while analyze project `{ProjectGuid}`